config/7.0/obdemo-bank/ig/routes/routes-service/10-ob-account-consent.json

```diff
@@ -56,25 +71,11 @@
                 "delegate": {
                   "comment":"resolve access tokens and retrieve metadata about the token. The endpoint typically returns the time until the token expires, the OAuth 2.0 scopes associated with the token, and potentially other information",
                   "name": "token-resolver-1",
-                  "type": "TokenIntrospectionAccessTokenResolver",
+                  "type": "StatelessAccessTokenResolver",
                   "config": {
-                    "endpoint": "https://&{identity.platform.fqdn}/am/oauth2/realms/root/realms/&{am.realm}/introspect",
```

Comment:
it's great to be able to achieve the functionality that we required without having to make an out of process call to AM